<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompetenceCell</class>
 <widget class="QWidget" name="CompetenceCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Domaine : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_domain"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Comp√©tences :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list_comps"/>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_comp"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>Ajouter</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="text">
        <string>Supprimer</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
